docs/supported/flockercontainers.rst

```diff
@@ -8,20 +8,16 @@ Running Flocker in Containers
 
     <div class="admonition labs">
         <p>This page describes one of our experimental projects, developed to less rigorous quality and testing standards than the mainline Flocker distribution. It is not built with production-readiness in mind.</p>
```

Comment:
Sooner or later we're going to have to create an official Flocker Docker image and at that time we can update the docs.